### PR TITLE
Added Recipe Model

### PR DESCRIPTION
**Pull request by Gontse**
Added recipe model

### CODE REVIEW

File: Recipe-App/Recipe-App/Models/Recipe.swift [hunk 1]
```diff
@@ -0,0 +1,22 @@
+//
+//  Recipe.swift
+//  Recipe-App
+//
+//  Created by Gontse Ranoto on 2021/06/11.
+//
+
+import Foundation
+
+struct Recipe {
```
Comment (MicahSphelele) [inline Recipe-App/Recipe-App/Models/Recipe.swift]: Can you extend Recipe with Identifiable and Codable

**Review by MicahSphelele — CHANGES_REQUESTED** [review]

File: Recipe-App/Recipe-App/Models/Recipe.swift (hunk 1 above)
Comment (Gontse) [reply Recipe-App/Recipe-App/Models/Recipe.swift]: Identifiable does not conform to codable...

Comment (MicahSphelele) [reply Recipe-App/Recipe-App/Models/Recipe.swift]: Okay that's weird.

Comment (MicahSphelele) [reply Recipe-App/Recipe-App/Models/Recipe.swift]: Okay re-request review so that I can approve.

**Review by MicahSphelele — APPROVED** [review]

**Pull request merged**